MetadataMigration/src/test/java/org/opensearch/migrations/ReplicaCountWithAZsTest.java

```diff
@@ -0,0 +1,78 @@
+package org.opensearch.migrations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+
+import org.opensearch.migrations.bulkload.framework.SearchClusterContainer;
+import org.opensearch.migrations.bulkload.framework.SearchClusterWithZoneAwarenessContainer;
+import org.opensearch.migrations.commands.MigrationItemResult;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@Tag("isolatedTest")
+public class ReplicaCountWithAZsTest extends BaseMigrationTest{
+
+    private static final SearchClusterContainer.ContainerVersion SOURCE_VERSION = SearchClusterContainer.ES_V7_10_2;
+    private static final SearchClusterContainer.ContainerVersion TARGET_VERSION = SearchClusterContainer.OS_LATEST;
```

Comment:
Its strange that this is unused, maybe the zoned search cluster is a static class for only this test case